Sources/StreamChat/WebSocketClient/RetryStrategy_Tests.swift

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChat
+import XCTest
+
+class RetryStrategy_Tests: XCTestCase {
+    var strategy: DefaultRetryStrategy!
+    
+    override func setUp() {
+        super.setUp()
+        strategy = DefaultRetryStrategy()
```

Comment:
Lets clear this in the tearDown to avoid any hanging references in the memory.

[Read more here](https://qualitycoding.org/xctestcase-teardown/)

Reply:
✅ Updated